database/payment.go

```diff
@@ -3,7 +3,8 @@ package database
 import (
 	"context"
 	"fmt"
-	"memberserver/mail"
+
+	//"memberserver/mail"
```

Comment:
Remove it. No need to comment this out since this will automatically be added again if you have go tools installed

Reply:
This file will be cleaned up in my next PR to resolve #101.